api/layer/object_cache.go

```diff
@@ -0,0 +1,111 @@
+package layer
+
+import (
+	"sync"
+	"time"
+
+	cid "github.com/nspcc-dev/neofs-api-go/pkg/container/id"
+)
+
+/*
+	This is an implementation of a cache for ListObjectsV2 which we return to users by ContinuationToken.
+
+	The cache consists of a map with data and a timer.
+	The map has a key: (access_key from AccessBox) + container_id and a value: list of objects with creation time.
+	The timer every timerDefaultTickerTime checks the map with caches and removes the expired caches.
```

Comment:
Is it too expensive if we run `time.AfterFunc()` on each key with a removal function? Maybe it will be more profitable if this cache is empty most of the time.

Reply:
Thanks, time.After() is not too expensive, changed a way to clean records of the cache.